front/src/common/authorization/components/GrantsManager.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+
+import { ChevronDown, ChevronUp } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+
+import type { Grant, ResourceType } from 'common/api/mock/mockEditoastApi';
+import { GrantsLabel } from 'modules/infra/consts';
+import { capitalizeFirstLetter } from 'utils/strings';
+
+import GrantsManagerSubjects from './GrantsManagerSubjects';
+
+type GrantsManagerProps = {
+  resourceId: number;
+  resourceType: ResourceType;
+  userGrant: Grant;
+  resourceGrants?: {
+    [grant: string]: string[];
+  };
+};
+
+const GrantsManager = ({
+  resourceId,
+  resourceType,
+  userGrant,
+  resourceGrants = {},
+}: GrantsManagerProps) => {
+  const { t } = useTranslation('grantManagement');
+  const [displayGrantSection, setDisplayGrantSection] = useState(false);
+
+  return (
+    <div
+      className="grant-manager"
+      role="button"
+      tabIndex={0}
+      // We need that to prevent the modal to close when clicking on this section
+      onClick={(e) => e.stopPropagation()}
+    >
+      <div className="grant-manager-header">
+        <span className="user-infra-grant">
+          {t('yourGrant', {
+            grant: capitalizeFirstLetter(t(`grants.${GrantsLabel[userGrant]}`)),
+          })}
+        </span>
+        <button
+          type="button"
+          onClick={() => {
+            setDisplayGrantSection(!displayGrantSection);
+          }}
+        >
+          <span className={displayGrantSection ? 'blue-label' : 'grey-label'}>
+            {displayGrantSection ? t('actions.collapse') : t('actions.details')}
+            {displayGrantSection ? <ChevronUp /> : <ChevronDown />}
+          </span>
```

Comment:
What do you think of that ?
```suggestion
          {displayGrantSection ? (
            <span className="blue-label">
              {t('actions.collapse')}
              <ChevronUp />
            </span>
          ) : (
            <span className="grey-label">
              {t('actions.details')}
              <ChevronDown />
            </span>
          )}
```